packages/ui/theme/components/Button.ts

```diff
@@ -49,12 +50,14 @@ export const ButtonStyleConfig: ComponentStyleConfig = {
       borderWidth: 2,
       borderColor: mode('ecru', 'ecru')(props),
       _hover: {
-        bg: mode('ecru80', 'ecru80')(props),
+        textDecoration: 'unset',
+        bg: mode('ecru', 'ecru')(props),
         color: mode('raisinBlack', 'raisinBlack')(props),
       },
     }),
     filter: (props) => ({
       ...theme.components.Button.variants?.solid,
+      height: '52px',
```

Comment:
Not going to claim that I understand how any of these work, but the output looks good :)